### PR TITLE
Add EPHEMERAL_TEST  variable for e2e test in epemeral cluster.

### PR DESCRIPTION
**Pull request by smoshiur1237**
EPHEMERAL_TEST variable is added to enable the CAPM3 e2e test in ephemeral cluster.  
The following jenkins pipeline [PR](https://gerrit.nordix.org/c/infra/cicd/+/12728) is created to have the CI job and use EPHEMERAL_TEST variable .

### CODE REVIEW

File: jenkins/scripts/files/run_integration_tests.sh [hunk 1]
```diff
@@ -13,6 +13,7 @@ export IMAGE_OS
 export DEFAULT_HOSTS_MEMORY
 export NUM_NODES
 export UPGRADE_TEST
+export EPHEMERAL_TEST
```
Comment (fmuyassarov) [inline jenkins/scripts/files/run_integration_tests.sh:16]: it is already in the vars.sh which is sourced in L8. I think this is redundant.

File: jenkins/scripts/files/run_integration_tests.sh [hunk 2]
```diff
@@ -36,6 +37,11 @@ then
   unset UPGRADE_TEST
 fi
 
+if [ "${EPHEMERAL_TEST}" == "null" ]
+then
+  unset EPHEMERAL_TEST
```
Comment (fmuyassarov) [inline jenkins/scripts/files/run_integration_tests.sh:42]: or rather set to default value `false` as in CAPM3?

Comment (smoshiur1237) [reply jenkins/scripts/files/run_integration_tests.sh:42]: I followed the same way as `UPGRADE_TEST`  and `NUM_NODES` variable.

File: jenkins/scripts/files/run_integration_tests.sh (hunk 1 above)
Comment (smoshiur1237) [reply jenkins/scripts/files/run_integration_tests.sh:16]: I understand, but follow the same pattern like other variables such as UPGRADE_TEST

Comment (Rozzii) [reply jenkins/scripts/files/run_integration_tests.sh:16]: Yeah this section is a bit funky with the exports, but I am not sure if you should fix it here or in an other PR though.

Comment (fmuyassarov) [reply jenkins/scripts/files/run_integration_tests.sh:16]: Let's tackle this on a follow up PR @smoshiur1237 
/approve

**Pull request merged**